src/latexify_recipes.jl

```diff
@@ -42,9 +42,13 @@ function _toexpr(O; canonicalize=true)
     else
         !istree(O) && return O
     end
-
     op = operation(O)
     args = arguments(O)
+    
+    if (op===(*)) && (args[1] == -1)
+	return Symbol("-$(args[2])")
```

Comment:
```suggestion
        arg_mul = Expr(:call, :(*), _toexpr(args[2:end]; canonicalize=canonicalize)...)
        return Expr(:call, :(-), arg_mul)
```

Reply:
cool...now I understand more about Julia expression. I notice there is a `latexify.jl` which is not included in the `runtest.jl`, any reason for the exclusion? I am thinking just adding more tests in that particular file

Reply:
Nope, that was an oversight. If you could add that file to `runtests.jl` that would be great.